src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -334,7 +334,7 @@ private FunctionPointerMethodSymbol(CallingConvention callingConvention, Immutab
             RefCustomModifiers = CSharpCustomModifier.Convert(retInfo.RefCustomModifiers);
             CallingConvention = callingConvention;
             ReturnTypeWithAnnotations = returnType;
-            RefKind = getRefKind(retInfo, RefCustomModifiers, RefKind.RefReadOnly);
+            RefKind = getRefKind(retInfo, RefCustomModifiers, RefKind.RefReadOnly, RefKind.Ref);
```

Comment:
>RefKind.Ref); [](start = 83, length = 13)

I didn't understand this. Why are we mapping `OutAttribute` to `RefKind.Ref`?

Reply:
> I didn't understand this. Why are we mapping ```OutAttribute``` to ```RefKind.Ref```?

What would you expect us to do here?

---
In reply to: [441161001](https://github.com/dotnet/roslyn/pull/45220#discussion_r441161001) [](ancestors = 441161001)

Reply:
I'm not sure, that's why I'm asking for clarification. Why are we mapping OutAttribute to RefKind.Ref?

Reply:
> I'm not sure, that's why I'm asking for clarification. Why are we mapping OutAttribute to RefKind.Ref?

What do you think is a better option?

---
In reply to: [441164767](https://github.com/dotnet/roslyn/pull/45220#discussion_r441164767) [](ancestors = 441164767)